brdnet/download_categorized_data.ipynb

```diff
@@ -0,0 +1,4362 @@
+      "\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: 2019-07-18 14:18:54 downloading file rse_gene.Rdata to ../data/recount/SRP028291\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: trying URL 'http://duffel.rail.bio/recount/v2/SRP028291/rse_gene.Rdata'\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning:  length 4827890 bytes (4.6 MB)\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: 2019-07-18 14:18:56 downloading file rse_gene.Rdata to ../data/recount/ERP006650\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: trying URL 'http://duffel.rail.bio/recount/v2/ERP006650/rse_gene.Rdata'\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning:  length 5567552 bytes (5.3 MB)\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: downloaded 5.3 MB\n",
+      "\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: 2019-07-18 14:18:57 downloading file rse_gene.Rdata to ../data/recount/SRP064138\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: trying URL 'http://duffel.rail.bio/recount/v2/SRP064138/rse_gene.Rdata'\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning:  length 2921751 bytes (2.8 MB)\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: 2019-07-18 14:18:59 downloading file rse_gene.Rdata to ../data/recount/SRP041531\n",
+      "\n",
+      "  warnings.warn(x, RRuntimeWarning)\n",
+      "/home/ben/anaconda3/envs/brdnet/lib/python3.7/site-packages/rpy2/rinterface/__init__.py:146: RRuntimeWarning: trying URL 'http://duffel.rail.bio/recount/v2/SRP041531
```

Comment:
Does this somehow pass the Python list to R?

Reply:
Yep, rpy lets you pass lists and dataframes between R and python